docs/advanced-examples.md

```diff
@@ -1 +1,344 @@
+
+As a reminder, at the end of the pipeline your data needs to adhere to the following format. Extra fields are ignored. For more information, see the [aggregator](/aggregator) documentation.
+
+```json
+{
+    "project": "<some unique identifier for the project, such as the repo name or package>",
+    "version": "<some way to represent the version the code is running on, usually a git hash>",
+    "location": {
+        "file": "<the file in which this log entry occured, relative to the root of the code>",
+        "line": "<the line in which the log happened, can be a string or a number>"
+    },
+    "severity": "<some way to represent the severity, recommended to be a standard severity but free form>",
+    "timestamp": "<an ISO 8601-parsable timestamp that represents the time at which this log occurred>"
+}
+```
+
+As such, you will need to engineer your pipeline in a way that the plugins transform your input format into this final format. This document has a list of commonly used plugins and example scenarios in which they can be used.
+
+If you need a way to do something not listed here, we've made it easy to create custom plugins! See our documentation on [creating a new pipeline plugin in Java/Kotlin](/docs/writing-java-kotlin-plugin.md) or [creating a new pipeline plugin in a different language](/docs/writing-custom-plugin.md).
+
+## Document structure
+
+This document has two sections. First, there is a list of plugins and example scenarios in which they may be used. These each describe the plugins workings on a simple level. The [plugin index](#plugin-index) has a full list of the plugins and the scenarios in which they may be used.
+
+Secondly, the (example pipelines)[#example-pipelines] section contains a set of example input payloads and the recommended set of plugins that can be used to transform them to the required output format. They are meant to be abridged versions of the [main Hyperion pipeline tutorial](/docs/hyperion-setup.md), and as such do not detail full configuration files but rather a list of plugins and references to their appropriate documentation.
+
+## Plugin Index
+
+I want to...
+
+- [Inspect the current messages going through the pipeline](#plugin-printer)
+- [Measure the rate of messages going through the pipeline](#plugin-rate)
+- [Rename or move a field](#plugin-renamer)
+- [Add a statically known value to the payload](#plugin-adder)
+- [Extract string values from a field into their own fields](#plugin-extractor)
+- [Transform a Java package name to a file path](#plugin-pathextractor)
+- [Attach a version tag based on the latest commit in a Git repository](#plugin-versiontracker)
```

Comment:
Same here:
`[Load balance multiple instances of a single plugin to increase pipeline throughput](#plugin-version-tracker)`